inflammation-analysis.py

```diff
@@ -20,14 +20,14 @@ def main(args):
 
     for filename in infiles:
         inflammation_data = models.load_csv(filename)
-
         view_data = {
             'average': models.daily_mean(inflammation_data),
             'max': models.daily_max(inflammation_data),
             'min': models.daily_min(inflammation_data)
         }
+     develop
 
-        views.visualize(view_data)
+     views.visualize(view_data)
```

Comment:
I think we did this last week, but maybe it would be a good idea to return just the data instead of a graph. a new function could be made that only plots the data.

Reply:
adjusted